addons/hive.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: install hive
+  hosts: frontendnodes
+  become: yes
+  roles:
+    - role: java
+    - role: mysql
+      when: HIVE_METASTORE == 'mysql'
```

Comment:
After reading through the hive role it appears to require mysql.
Is that correct?
If so, you can delete the `when` clause for the `role: mysql`